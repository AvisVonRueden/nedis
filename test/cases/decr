*3
$3
SET
$1
n
$2
15
-------
*2
$4
DECR
$1
n
-------
*2
$4
DECR
$1
n
-------
*2
$4
DECR
$3
num
-------
*2
$4
DECR
$3
num
-------
*2
$4
DECR
$3
num
