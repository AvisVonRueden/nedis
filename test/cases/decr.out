+OK
:14
:13
